plugins/labhub.py

```diff
@@ -302,11 +302,27 @@ def difficulty_level(user, iss):
                 elif self.GH3_ORG.is_member(user):
                     return True
 
+            @register_check
+            def block_gci_issue_assignment(user, iss):
+                """
+                True if the issue is not labelled with 'initiatives/gci'.
+                False if the issue has been labelled with 'initiatives.gci'.
+                """
+                if 'initiatives/gci' in iss.labels:
+                    return False
+                else:
+                    return True
+
         def eligible(user, iss):
+            """
+            True if all checks are passed.
+            False if any check(s) is/are failed.
+            """
+            el = True
             for chk in checks:
-                if not chk(user, iss):
-                    return False
-            return True
+                # el evaluates to False if any check is failed.
+                el = el and chk(user, iss)
+            return el
```

Comment:
This change to the `eligible` method should be done in a new commit, though I'm not actually sure if I like this or the original better. This is definitely shorter, but also seems to be a little more diluted.